torchx/cli/cmd_log.py

```diff
@@ -101,12 +101,14 @@ def get_logs(
     if len(path) == 4:
         replica_ids = [(role_name, int(id)) for id in path[3].split(",") if id]
     else:
-        for i in range(10):
+        display_waiting = True
+        while True:
             status = runner.status(app_handle)
             if status and is_started(status.state):
                 break
-            if i == 0:
-                logger.info("Waiting for app to start before logging...")
+            elif display_waiting:
+                logger.info("Waiting for app state response before fetching logs...")
```

Comment:
since this is in the CLI code, rather than printing multiple log lines with the same message, why not print (to stdout) with a carriage-return (`\r`)?

Reply:
It should only display once -- display_waiting is immediately set to false. (I lost the iteration index while changing to the while loop)